dhis-2/dhis-web-api/src/main/resources/openapi/EnrollmentsExportController.md

```diff
@@ -53,73 +55,75 @@ excluded by default.
 
 `<orgUnit1-uid>[,<orgUnit2-uid>...]`
 
-Get enrollments owned by given `orgUnits` relative to the `orgUnitMode`.
+Get enrollments owned by given organisation units in `orgUnits` parameter relative to the 
+`orgUnitMode` parameter.
 
 - When `orgUnitMode=SELECTED` - or no `orgUnitMode` is given (default) - the enrollments owned by
   the `orgUnits` are returned.
 - When `orgUnitMode=CHILDREN` the enrollments owned by the `orgUnits` or by the `orgUnits` direct
-  children is returned.
+  children are returned.
 - When `orgUnitMode=DESCENDANTS` the enrollments owned by the `orgUnits` or any of its descendants
   are returned.
 - When `orgUnitMode=ALL`, `orgUnitMode=CAPTURE` or `orgUnitMode=ACCESSIBLE` the `orgUnits` parameter
   is not allowed.
 
 ### `*.parameter.EnrollmentRequestParams.orgUnitMode`
 
-Get enrollments using given organisation unit selection mode.
+Get enrollments using the given organisation unit selection mode.
 
-- When `orgUnitMode=SELECTED`, `orgUnitMode=CHILDREN` or `orgUnitMode=DESCENDANTS`, the `orgUnit`
-  parameter is
-  required to specify which enrollments to return.
+- When `orgUnitMode=SELECTED`, `orgUnitMode=CHILDREN` or `orgUnitMode=DESCENDANTS`, the `orgUnits`
+  parameter is required to specify which enrollments to return.
 - When `orgUnitMode=ALL` enrollments will be downloaded irrespective of the organization unit they
-  are
-  owned by. To use this parameter, the user needs the `Search Tracked entity in all org units`
+  are owned by. To use this parameter, the user needs the `Search Tracked entity in all org units`
   authority.
-- When `orgUnitMode=ACCESSIBLE` enrollments owned by any org unit in the users capture scope will be
-  returned.
-- When `orgUnitMode=CAPTURE` enrollments that has an enrollment org unit in the users capture scope
-  will
-  be returned.
+- When `orgUnitMode=ACCESSIBLE` enrollments owned by any organisation unit in the user's capture 
+  scope will be returned.
+- When `orgUnitMode=CAPTURE` enrollments that have an enrollment organisation unit in the user's 
+  capture scope will be returned.
 
 ### `*.parameter.EnrollmentRequestParams.program`
 
-Get enrollments enrolled in the given tracker program.
+Get enrollments enrolled in the given tracker program. Only enrollments related to the specified program will be returned.
 
 ### `*.parameter.EnrollmentRequestParams.status`
 
-Get enrollments in the given status.
+Get enrollments in the given status. Valid statuses are ACTIVE, COMPLETED, and CANCELLED.
 
 ### `*.parameter.EnrollmentRequestParams.programStatus`
 
 Get enrollments in the given status.
 
 **DEPRECATED as of 2.42:** Use parameter `status` instead.
 
-See `status` for details.
+See `status` for details on valid statuses.
 
 ### `*.parameter.EnrollmentRequestParams.trackedEntityType`
 
-Get enrollments of tracked entities of the given type.
+Get enrollments of tracked entities of the given type. Only enrollments associated with tracked 
```

Comment:
I think these additional sentences don't add value just more text. The same applies to all other parameters with similar changes in this PR. If the first sentence is not clear lets change that one.